api/card2k/exchange_card.py

```
@@ -0,0 +1,128 @@
+        url = f"{self.provider}/chargingws/v2/check-api"
+        try:
+            response = requests.get(url, json=payload)
+            response.raise_for_status()
+            data = response.json()
+            if 'status' not in data:
+                return False
+            return data["status"] == 'active'
+        except requests.RequestException as e:
+            logger.error(f"[API Card2K Exchange-Card] Lỗi hàm get_fee: {e}")
+            return None
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)